pom.xml

```diff
@@ -211,13 +162,116 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>migrate-db</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
```

Comment:
Interesting.  I'm surprised that the profile doesn't restrict all those plugins.  Unless they're all there only to run flyway.